<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clsMainDialog</class>
 <widget class="QDialog" name="clsMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>扫描温度曲线</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myIcon.qrc">
    <normaloff>:/Icon/Icons/wk.ico</normaloff>:/Icon/Icons/wk.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog
{
	
	font: 75 12pt &quot;楷体&quot;;
	background-color: rgb(148, 148, 248);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnTrig">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>开始
测试</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/single.png</normaloff>:/Icon/Icons/single.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnMeasStup">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>参数
设置</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/instrument.png</normaloff>:/Icon/Icons/instrument.png</iconset>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnTraceSetup">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>扫描
设置</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/trace.png</normaloff>:/Icon/Icons/trace.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAutoFit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>自动
适应</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/autoFit.png</normaloff>:/Icon/Icons/autoFit.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbTcType">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;楷体&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>J</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>K</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>S</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>T</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>E</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbType">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;楷体&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>℃</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>℉</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnData">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>保存
数据</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/data.png</normaloff>:/Icon/Icons/data.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveImg">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>36</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;楷体&quot;;</string>
         </property>
         <property name="text">
          <string>保存
图像</string>
         </property>
         <property name="icon">
          <iconset resource="myIcon.qrc">
           <normaloff>:/Icon/Icons/HILowLimit.png</normaloff>:/Icon/Icons/HILowLimit.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="plotWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>536</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>36</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>18</number>
        </property>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rbtnBias2">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;楷体&quot;;
background-color: rgb(0, 255, 0);</string>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="rbtnBias4">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;楷体&quot;;
background-color: rgb(127, 0, 127);</string>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="rbtnBias1">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;楷体&quot;;
background-color: rgb(255, 0, 0);</string>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="rbtnBias3">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;楷体&quot;;
background-color: rgb(0, 0, 255);</string>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>314</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="rbtnBias5">
          <property name="styleSheet">
           <string notr="true">font: 75 10pt &quot;楷体&quot;;
background-color: rgb(0, 255, 255);</string>
          </property>
          <property name="text">
           <string>1.00A</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myIcon.qrc"/>
 </resources>
 <connections/>
</ui>
